blocks/v2-columns/v2-columns.js

```diff
@@ -0,0 +1,31 @@
+import { readBlockConfig } from '../../scripts/lib-franklin.js';
+import { getTextLabel, createElement } from '../../scripts/common.js';
+
+const ctaText = getTextLabel('learn more');
+
+export default async function decorate(block) {
+  const picture = block.querySelector('picture');
+  const blockConfig = readBlockConfig(block);
+  const [, eyebrow, headline, body, link] = Object.values(blockConfig);
+
+  const columnsImage = createElement('div', { classes: 'v2-columns-image' });
+  const columnsText = createElement('div', { classes: 'v2-columns-content' });
+
+  const eyebrowElmt = createElement('p', { classes: 'v2-columns-content__eyebrow' });
+  eyebrowElmt.textContent = eyebrow;
+  const headlineElmt = createElement('h1', { classes: 'v2-columns-content__headline' });
```

Comment:
We will break the semantic rules using `h1` always. We need the editor to decide which tag they want to use.

Reply:
Plus it should be only one h1 per document, and this should be provided using Hero block or by editor using free-block content with Heading1 MS Word style.

Reply:
this is something I forgot to ask... I agree with you but in figma it was an H1... should I choose then which other H to use here?

Reply:
What is said in Figma is the style of the heading (h1), but not the tag. Tag needs to be defined by the editor.